test/unit/querier.test.ts

```diff
@@ -10,8 +10,6 @@ describe("Querier", () => {
             apiBasePath: new NormalisedURLPath("/"),
             apiDomain: new NormalisedURLDomain("http://api.example.com"),
             appName: "test",
-            websiteBasePath: new NormalisedURLPath("/"),
-            websiteDomain: new NormalisedURLDomain("http://example.com"),
         });
```

Comment:
Shouldn't this whole file be moved to web-js as well?

Reply:
Done, refer to: https://github.com/supertokens/supertokens-web-js/pull/25